packs/ansible/actions/galaxy.install.yaml

```diff
@@ -0,0 +1,52 @@
+---
+name: "galaxy.install"
+description: "Download & Install role from ansible galaxy"
+runner_type: "local-shell-script"
+entry_point: "ansible_galaxy.py"
+enabled: true
+parameters:
+  kwarg_op:
+    description: "Lock operator type to '--'"
+    type: string
+    default: "--"
+    immutable: true
+  sudo:
+    description: "Lock sudo, the behavior is controlled by ansible 'become_' options"
+    type: boolean
+    default: true
+    immutable: true
+  timeout:
+    description: "Action timeout in seconds. Action will get killed if it doesn't finish in timeout seconds"
+    type: integer
+    default: 300
+  cwd:
+    description: "Working directory where the command will be executed in"
+    type: string
+  action:
+    description: "Action to use"
+    type: string
+    position: 0
+    default: install
+    immutable: true
+  roles:
```

Comment:
Ideally, we would know the required parameters before the run time, just by introspection the metadata (this way we can provide better validation error messages, generate more user-friendly UI forms, etc.), but we don't allow users to specify mutually exclusive parameters in the metadata file right now so your approach is fine in this case.